plugins/typo_squatter/komand_typo_squatter/util/utils.py

```diff
@@ -43,12 +78,12 @@ def score_domain(domain):
             score += 10
 
     # Testing keywords
-    for word in keywords.keys():
-        if word in domain:
-            score += keywords[word]
+    for word in keywords.items():
+        if word[0] in domain:
+            score += word[1]
 
-    # Higer entropy is kind of suspicious
-    score += int(round(entropy.shannon_entropy(domain) * 50))
+    # Higher entropy is kind of suspicious
+    score += int(round(entropy(domain) * 10))
```

Comment:
Any reason that 50 was reduced to 10 here? What are these magic numbers? Established constants or are we just kind of picking a number that generally makes things work?

Reply:
The entropy library is no longer available, so I adapted this number to the new Shannon entropy method used, based on the code in the link above.